docs/tasks/US-5-Get-Command-Reconstruction.md

```
@@ -0,0 +1,60 @@
+  "checksum": { "alg": "sha256", "value": "aabbcc..." }
+}
+```
+- It then iterates through the `chunks` array in ascending `index` order, verifying indices are contiguous and sizes sum to `total_size` (when provided).
+- For each chunk, it fetches the corresponding Git blob using its `blob_oid`.
+- The content of the blobs are written to the output stream in the correct order (by chunk index `i`).
+- The entire process is transparent to the user, who simply receives the full, reconstructed file content.
+
+## 3. Test Plan
+
+- **Integration Test (Round Trip):** `set` a large file so it chunk-manifests, then `get` the same key and compare checksums.
+- **Unit Test (Nominal Reconstruction):** Provide a mock manifest and chunk provider; assert chunks are requested sequentially by `index` and output order matches.
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)